frontend/awx/resources/templates/WorkflowJobTemplateForm.tsx

```diff
@@ -0,0 +1,212 @@
+  const onSubmit: PageFormSubmitHandler<WorkflowJobTemplateFormType> = async (
+    values: WorkflowJobTemplateFormType
+  ) => {
+    const {
+      arrayedJobTags,
+      arrayedSkipTags,
+      summary_fields: { labels },
+    } = values;
+
+    let jobTags = '';
+    let skipTags = '';
+    if (arrayedJobTags?.length) {
+      jobTags = stringifyTags(arrayedJobTags);
+    }
+    if (arrayedSkipTags.length) {
+      skipTags = stringifyTags(arrayedSkipTags);
+    }
+
+    await requestPatch<WorkflowJobTemplate>(`/api/v2/workflow_job_templates/${id}/`, {
+      ...values,
+      job_tags: jobTags ?? '',
+      skip_tags: skipTags ?? '',
+      webhook_credential: values.summary_fields.webhook_credential?.id,
+    });
+    (cache as unknown as { clear: () => void }).clear?.();
+    const promises = [];
+
+    promises.push(submitLabels(workflowJobTemplate as WorkflowJobTemplate, labels?.results));
+    navigate(RouteObj.WorkflowJobTemplateDetails.replace(':id', `${id}`.toString()));
+  };
+  if (error instanceof Error) {
+    return <AwxError error={error} handleRefresh={refresh} />;
+  }
+  if (isLoading) return <LoadingPage />;
+  return (
+    <PageLayout>
+      <PageHeader
+        title={t('Edit Job Template')}
+        breadcrumbs={[
+          { label: t('Templates'), to: RouteObj.Templates },
+          { label: t('Edit Job Template') },
+        ]}
+      />
+      <PageForm<WorkflowJobTemplateFormType>
+        submitText={t('Save workflow job template')}
+        onSubmit={onSubmit}
+        onCancel={() => navigate(-1)}
+        defaultValue={defaultValues}
+      >
+        <WorkflowJobTemplateInputs workflowJobTemplate={defaultValues} />
+      </PageForm>
+    </PageLayout>
+  );
+}
+export function CreateWorkflowJobTemplate() {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const postRequest = usePostRequest<WorkflowJobTemplate>();
+
+  const defaultValues: {
+    arrayedSkipTags: { value: string; label: string }[];
+    arrayedJobTags: { value: string; label: string }[];
+  } = useMemo(
+    () => ({
+      arrayedSkipTags: [{ value: '', label: '' }],
+      arrayedJobTags: [{ value: '', label: '' }],
+    }),
+    []
+  );
+  const onSubmit: PageFormSubmitHandler<WorkflowJobTemplateFormType> = async (values) => {
+    const {
+      arrayedJobTags,
+      arrayedSkipTags,
+      summary_fields: { labels, webhook_credential },
+    } = values;
+    let jobTags = '';
+    let skipTags = '';
+    if (values?.arrayedJobTags?.length) {
+      jobTags = stringifyTags(arrayedJobTags);
+    }
+    if (values?.arrayedSkipTags.length) {
+      skipTags = stringifyTags(arrayedSkipTags);
+    }
+
+    const template = await postRequest(`/api/v2/workflow_job_templates/`, {
+      ...values,
+      job_tags: jobTags,
+      skip_tags: skipTags,
+      summary_fields: values.summary_fields,
+      webhook_credential: webhook_credential?.id ? webhook_credential?.id : null,
+    });
```

Comment:
We don't need to send `summary_fields` on workflow template creation. I think the API is expecting a payload that looks something like this: 
```
interface WFCreation {
  allow_simultaneous: boolean;
  ask_inventory_on_launch: boolean;
  ask_labels_on_launch: boolean;
  ask_limit_on_launch: boolean;
  ask_scm_branch_on_launch: boolean;
  ask_skip_tags_on_launch: boolean;
  ask_tags_on_launch: boolean;
  ask_variables_on_launch: boolean;
  description: string;
  extra_vars: string;
  inventory: number;
  job_tags: string;
  limit: string;
  name: string;
  organization: number;
  scm_branch: string;
  skip_tags: string;
  survey_enabled: boolean;
  webhook_credential: number;
  webhook_service: 'github' | 'gitlab' | '';
}
```